CinnVIIStarkMenu@NikoKrause/po/CinnVIIStarkMenu.pot

```diff
@@ -108,11 +108,11 @@ msgid "Menu"
 msgstr ""
 
 #: 3.2/applet.js:3260 2.6/applet.js:3280
-msgid "This file is no longer available"
+msgid "This file can't be found"
```

Comment:
It could say "This file can't be found right now" instead to make it more understandable